chain-impl-mockchain/Cargo.toml

```diff
@@ -11,8 +11,8 @@ edition = "2018"
 license = "MIT OR Apache-2.0"
 
 [dependencies]
-serde = { version = "^1.0", optional = true }
-serde_derive = { version = "^1.0", optional = true }
+serde = { version = "1.0", features = ["derive"] }
+serde_derive = { version = "^1.0" }
```

Comment:
that shouldn't be modified

Reply:
I needed to do that to derive Serialize/Deserialize for some of the data types. If you know a better way, please do change that.